admin/app/models/certification-details.js

```diff
@@ -74,4 +83,8 @@ export default class CertificationDetails extends Model {
       };
     },
   });
+
+  #includePixPlusCompetences(answersByCompetence) {
```

Comment:
```suggestion
  #shouldIncludePixPlusCompetences(answersByCompetence) {
```
On pourrait aussi utiliser directement l'implém sans faire de méthode vu la "complexité"

Reply:
C'est simple mais pas explicite. Du coup on a fait une petite methode